docs/source/api/turbulence.rst

```diff
@@ -0,0 +1,11 @@
+***********************************
+Turbulence Time Series Calculations
+***********************************
+
+
+:mod:`metpy.calc.turbulence`
+========================
```

Comment:
Yep - it did. I also get a warning about not having the directory

docs/source/_static